docs/standard/frameworks.md

```diff
@@ -1,56 +1,293 @@
+--- | --- | ---
+.NET Framework | net | net11
+ | | net20
+ | | net35
+ | | net40
+ | | net403
+ | | net45
+ | | net451
+ | | net452
+ | | net46
+ | | net461
+ | | net462
+.NET Core | netcore | netcore [netcore45]
+ | | netcore45 [win, win8]
+ | | netcore451 [win81]
+ | | netcore50
+.NET MicroFramework | netmf | netmf
+Windows | win | win [win8, netcore45]
+ | | win8 [netcore45, win]
+ | | win81 [netcore451]
+ | | win10 (not supported by Windows 10 platform)
+Silverlight | sl | sl4
+ | | sl5
+Windows Phone | wp | wp [wp7]
+ | | wp7
+ | | wp75
+ | | wp8
+ | | wp81
+ | | wpa81
+Universal Windows Platform | uap | uap [uap10.0]
+ | | uap10.0
+.NET Standard | netstandard | netstandard1.0
+ | | netstandard1.1
+ | | netstandard1.2
+ | | netstandard1.3
+ | | netstandard1.4
+ | | netstandard1.5
+ | | netstandard1.6
+.NET Core App | netcoreapp | netcoreapp1.0
 
-**Referring to a product**
+## Deprecated frameworks
 
-You can refer to a .NET platform or runtime.
+The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
 
-- ".NET Framework 4.6.1"
+Deprecated framework | Replacement
+--- | ---
+aspnet50 | netcoreapp
+aspnetcore50 | 
+dnxcore50 | 
+dnx | 
+dnx45 | 
+dnx451 | 
+dnx452 | 
+dotnet | netstandard
+dotnet50 | 
+dotnet51 | 
+dotnet52 | 
+dotnet53 | 
+dotnet54 | 
+dotnet55 | 
+dotnet56 | 
+winrt | win
 
-**Referring to a Framework**
+## Precedence
 
-You can refer to a framework or targeting of a framework using long- or short-forms of the TFM. Both are equally valid in the general case.
+A number of frameworks are related to and compatible with one another but not necessarily equivalent:
 
-- `.NETFramework,Version=4.6.1`
-- `net461`
+Framework | Can use
+--- | ---
+uap (Universal Windows Platform) | win81
+| | wpa81
+| | netcore50
+win (Windows Store) | winrt
+| | winrt45
 
-**Referring to a family of Frameworks**
+## .NET Standard
 
-You can refer to a family of frameworks using long- or short-forms of the framework ID. Both are equally valid in the general case.
+The [.NET Standard](https://github.com/dotnet/standard) simplifies references between binary-compatible frameworks, allowing a single target framework to reference a combination of others. For background, see the [.NET Platform Guide](https://docs.microsoft.com/dotnet/articles/standard/index).
 
-- `.NETFramework`
-- `net`
+The [NuGet Tools Get Nearest Framework Tool](https://aka.ms/s2m3th) simulates the NuGet logic used for the selection of one framework from many available framework assets in a package based on the project's framework. To use the tool, enter one project framework and one or more package frameworks. Click the **Submit** button. The tool indicates if the package frameworks you list are compatible with the project framework you provide.
+
+For NuGet 3.3 and earlier, use the `dotnet` series of monikers. For NuGet 3.4 and later, use the `netstandard` moniker syntax.
+
+## Portable Class Libraries
```

Comment:
this one is a really long list and we don't even recommend people using it anymore. I think in this case we could link to the NuGet topic (https://docs.microsoft.com/nuget/schema/target-frameworks#portable-class-libraries)